examples/quickstart.yaml

```diff
@@ -46,6 +46,8 @@ metadata:
 data:
   config.yaml: |
     apiHost: https://api.honeycomb.io/
+    retryBufferSize: 1000
```

Comment:
Should we default to 0 to keep existing behavior, or is the retry logic critical enough that it should be used? I guess when we add it to the helm chart we could put in these default values but commented out...

Reply:
Yeah, we can set this to 0 for the default, and document it with recommended values, etc.